src/zope/testrunner/profiling.py

```diff
@@ -26,7 +25,7 @@
 try:
     import cProfile
     import pstats
-except ImportError:
+except ImportError: # pragma: no cover (Where could this fail?)
```

Comment:
A long time ago Linux distros used to split off `profile` and `pstats` modules into a [separate python-profile distro package](https://bugs.launchpad.net/ubuntu/+source/python-defaults/+bug/123755) for [licencing reasons](https://bugs.python.org/issue12417).  This was resolved in 2011.

Reply:
(Also, `cProfile` was not available in Python 2.4, and zope.testrunner *is* that old.)

Reply:
Anyway, I'm :+1: for not handling `ImportError`s from cProfile and pstats in 2017.